README.md

```diff
@@ -89,6 +89,32 @@ iex> D.div(D.new(31), D.new(2))
 #Decimal<15>
 ```
 
+### Comparisons
+
+Using compare operators (`<`, `=`, `>`) directly in two decimals may not return
+the correct result. Instead use comparison functions.
+
+```elixir
+iex> D.compare(D.new(-1), D.new(0))
+#Decimal<-1>
+iex> D.compare(D.new(0), D.new(-1))
+#Decimal<1>
+iex> D.compare(D.new(0), D.new(0))
+#Decimal<0>
+
+iex> D.cmp(D.new(-1), D.new(0))
```

Comment:
we've recently landed a change that will allow rewriting this as `D.cmp(-1, 0)` which I think is much cleaner. I'm happy to rewrite rest of the README after this PR lands @ericmj

Reply:
Yes, that sounds great 👍